include/seqan3/core/add_enum_bitwise_operators.hpp

```diff
@@ -34,10 +90,14 @@ namespace seqan3
 template <typename t>
 constexpr bool add_enum_bitwise_operators = false;
 
+//!\cond DEV
```

Comment:
Move this cond to line 77 (such that add_enum_bitwise_operators  is also DEV only)